configs/opencv-ovms/instance_segmentation_demo/python/instance_segmentation_demo.py

```diff
@@ -0,0 +1,276 @@
+                      help='Optional. Specify the target device to infer on; CPU or GPU is '
+                           'acceptable. The demo will look for a suitable plugin for device specified. '
+                           'Default value is CPU.')
+
+    common_model_args = parser.add_argument_group('Common model options')
+    common_model_args.add_argument('--labels', required=True,
+                                   help='Required. Path to a text file with class labels.')
+    common_model_args.add_argument('-t', '--prob_threshold', default=0.5, type=float,
+                                   help='Optional. Probability threshold for detections filtering.')
+    common_model_args.add_argument('--no_track', action='store_true',
+                                   help='Optional. Disable object tracking for video/camera input.')
+    common_model_args.add_argument('--show_scores', action='store_true',
+                                   help='Optional. Show detection scores.')
+    common_model_args.add_argument('--show_boxes', action='store_true',
+                                   help='Optional. Show bounding boxes.')
+    common_model_args.add_argument('--layout', type=str, default=None,
+                                   help='Optional. Model inputs layouts. '
+                                        'Format "[<layout>]" or "<input1>[<layout1>],<input2>[<layout2>]" in case of more than one input. '
+                                        'To define layout you should use only capital letters')
+
+    infer_args = parser.add_argument_group('Inference options')
+    infer_args.add_argument('-nireq', '--num_infer_requests', default=0, type=int,
+                            help='Optional. Number of infer requests')
+    infer_args.add_argument('-nstreams', '--num_streams', default='',
+                            help='Optional. Number of streams to use for inference on the CPU or/and GPU in throughput '
+                                 'mode (for HETERO and MULTI device cases use format '
+                                 '<device1>:<nstreams1>,<device2>:<nstreams2> or just <nstreams>).')
+    infer_args.add_argument('-nthreads', '--num_threads', default=None, type=int,
+                            help='Optional. Number of threads to use for inference on CPU (including HETERO cases).')
+
+    io_args = parser.add_argument_group('Input/output options')
+    io_args.add_argument('--loop', action='store_true',
+                         help='Optional. Enable reading the input in a loop.')
+    io_args.add_argument('-o', '--output',
+                         help='Optional. Name of the output file(s) to save.')
+    io_args.add_argument('-limit', '--output_limit', default=1000, type=int,
+                         help='Optional. Number of frames to store in output. '
+                              'If 0 is set, all frames are stored.')
+    io_args.add_argument('--no_show', action='store_true',
```

Comment:
NOTE: when we are running the benchmark script we want to use this --no_show parameter for better performance.

Reply:
Done